docs/index.md

```diff
@@ -21,10 +21,11 @@ Platform
 :fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 4.5.x on vSAN ReadyNodes | :fontawesome-solid-check:{ .green }
 :fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 4.4.x on vSAN ReadyNodes | :fontawesome-solid-check:{ .green }
 :fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 4.3.x on vSAN ReadyNodes | :fontawesome-solid-check:{ .green }
+:fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 5.0.x on VxRail          | :fontawesome-solid-check:{ .green }
```

Comment:
```suggestion
:fontawesome-solid-cloud: &nbsp; VMware Cloud Foundation 5.0.x on Dell VxRail     | :fontawesome-solid-check:{ .green }
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108